src/components/Properties/PropertyGroups.vue

```diff
@@ -55,9 +54,15 @@
 						{{ t('contacts', 'No results') }}
 					</span>
 				</NcSelect>
-				<p v-else>
-					{{ localValue.length === 0 ? t('contacts','None'): localValue.toString() }}
-				</p>
+				<div v-else>
+					<span v-if="localValue.length === 0">{{ t('contacts','None') }}</span>
+
+					<div class="group__list">
```

Comment:
```suggestion
					<div v-else class="group__list">
```
so we don't get an empty div in the dom